master/getting-started/bare-metal/installation/binary.md

```diff
@@ -1,124 +0,0 @@
----
-title: Binary install without package manager
```

Comment:
I am not sure we should delete this page. I had thought that we would retain this section but update it to instruct people on how to derive the binary from the image. We already have this content here https://docs.tigera.io/v2.2/getting-started/bare-metal/installation/binary

Reply:
The core team collectively agreed this during our planning meeting on Monday.  I was aware of that content at docs.tigera.io, but it's pretty complicated, and we suspect that this use case - using docker to extract the binary, but not using docker just to run the container - isn't actually significantly needed.

Reply:
I see. I just checked in with @kprabhak and he indicated that many of the hosts that require protection will not have Docker. So they can extract the binary from a machine with Docker but then put that binary on machines without it.

Reply:
OK, thanks @emanic .

@caseydavenport WDYT?  Are you OK with me pulling over the needed mods from https://docs.tigera.io/v2.2/getting-started/bare-metal/installation/binary, and with still doing that in the current sprint?